packages/cli/src/index.ts

```diff
@@ -251,6 +252,7 @@ const main = async () => {
 
   // Shared API `Client` instance for all sub-commands to utilize
   client = new Client({
+    agent: new ProxyAgent({ keepAlive: true }),
```

Comment:
Is there a way to add a test for this?

I suspect when we switch from `node-fetch` to native `fetch` in the future, that this agent won't work.

Reply:
> Is there a way to add a test for this?

Added a unit test in e9476c8352f2faf48a76bcac19883070c7758223.

> I suspect when we switch from `node-fetch` to native `fetch` in the future, that this agent won't work.

Looks like undici uses a different Agent format, so ya. We can cross that bridge when the time comes.